test/images/node-perf/tf-wide-deep/Dockerfile.ppc64le

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM python:3.6-slim-stretch
+
+RUN apt-get update && apt-get install -y wget time
+
+RUN wget -q ftp://ftp.unicamp.br/pub/ppc64el/ai_frameworks/tensorflow/tensorflow-1.12.0-cp36-cp36m-linux_ppc64le.whl
+RUN apt-get install -y gcc libhdf5-dev
+RUN pip install tensorflow-1.12.0-cp36-cp36m-linux_ppc64le.whl
+
+
+    
+
+RUN if [ $(uname -m) = "aarch64" ]; then \
+      pip install tensorflow-aarch64; \
+    elif [ $(uname -m) = "aarch64" ]; then \
```

Comment:
TBH, I am not familar with ppc64, but here should be different one because line 26 and line 28 conditions are the same now.

Reply:
this is my bad.! I'm not supposed to add this file itself. I'm still working on this file and will publish part of other PR!

Reply:
removed that file and force pushed the changes.